README.md

```diff
@@ -8,6 +8,8 @@ ZHA device handlers bridge the functionality gap created when manufacturers devi
 
 Custom quirks implementations for zigpy implemented as ZHA Device Handlers are a similar concept to that of [Hub-connected Device Handlers for the SmartThings Classics platform](https://docs.smartthings.com/en/latest/device-type-developers-guide/) as well that of [Zigbee-Herdsman Converters / Zigbee-Shepherd Converters as used by Zigbee2mqtt](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html), meaning they are virtual representation of a physical device that expose additional functionality that is not provided out-of-the-box by the existing integration between these platforms. See [Device Specifics](#Device-Specifics) for details.
 
+Note! The ZHA Device Handlers library is a standalone library and not dependent [Home Assistant](https://www.home-assistant.io), however a reference integration implementation of the ZHA Device Handlers library is available in [Home Assistant](https://www.home-assistant.io/) (a Python based open source home automation software) as part of its [ZHA integration component](https://www.home-assistant.io/integrations/zha/). The source code for that reference integration implementation is avalable in Home Assistant's "core" code under the "home-assistant/core/tree/dev/homeassistant/components/zha" directory.
```

Comment:
This isn't really true. there is no reference implementation in ZHA. All ZHA does is import the library... I think this is misleading. @Adminiuga thoughts?

Reply:
I think that you should refered to 'The Device Handlers library' and not the 'ZHA Device handlers'

Reply:
Reference implementation using zha device handlers library?

Reply:
By "reference implementation" I no not mean to imply that there is any kind of dependency on Home Assistant. I only meant that developers can look at Home Assistant's ZHA code as a reference in order to see how zigpy and zha-quirks can be used.

That terminology wording is used as a standard for these types of scenarios, see  https://en.wikipedia.org/wiki/Reference_implementation

"In the software development process, a reference implementation (or, less frequently, sample implementation or model implementation) is the standard from which all other implementations and corresponding customizations are derived. A reference implementation often accompanies a technical standard, and demonstrates what should be considered the "correct" behaviour of any other implementation of it."

Reply:
I know what a reference implementation is... this is not one. it's a dependency pure and simple. Again, HA only depends on and imports this library. That's it.

Reply:
Removed the phrasing "reference implementation".